### ISSUE TITLE
Getting different result with same seed and prompt

### ISSUE THREAD

**Issue by Yuhan-Ut** [alecjacobson/computer-graphics-text-to-image#11]
This is kind of wired that when I tried to run with same seed and same prompt, I am getting totally different result from yesterday, is this normal or something goes wrong?

**Comment by Yuhan-Ut** [comment]
Oh, My bad, seems this is already on another issue mentioned
emm, I am doing the open end one, and I just need one more (19/20), so can I just use the new one as the 20th pic?

**Comment by wenzhi-guo** [comment]
Yes, sometimes you do get a different result with the same prompt. Using the new one as the 20th image should be fine.
